prow/jobs/control-plane/control-plane-gke-provisioner-integration.yaml

```diff
@@ -0,0 +1,91 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+# Kubectl 1.15 and CLUSTER_VERSION=1.15
+base_image_k15: &base_image_k15
+  image: eu.gcr.io/kyma-project/test-infra/kyma-integration:v20200513-93bafa15-k8s1.15
```

Comment:
K8s 1.16 is already available in GKE standard channel so please use newer image template:
```
# Kubectl 1.16 and CLUSTER_VERSION=1.16
base_image_k16: &base_image_k16
  image: eu.gcr.io/kyma-project/test-infra/kyma-integration:v20200513-93bafa15-k8s1.16
```